wagtail/users/views/users.py

```diff
@@ -15,6 +15,7 @@
 from wagtail.admin.forms.search import SearchForm
 from wagtail.core import hooks
 from wagtail.core.compat import AUTH_USER_APP_LABEL, AUTH_USER_MODEL_NAME
+from wagtail.core.log_actions import log
```

Comment:
question: are images/documents outside of the scope of this PR?

Reply:
Yep - there's a bit of extra work involved for those, since we'd want the Site History report to only include images/documents in collections that the viewing user has 'choose' permission on, and in turn that means we need to store the collection ID as part of the log model for efficient filtering (and thus a dedicated MediaLogEntry model). Logging for images/documents wasn't part of Motley Fool's requirements, so hasn't been included in the development they're sponsoring.

(Having said that, the lack of logging on those models feels like a conspicuous absence when everything else has it, so I may well try and fit that in if there's time before 2.15rc1!)